### ISSUE TITLE
Differentiate between meida types (video or photo)

### ISSUE THREAD

**Issue by sparshy** [ugurozturk/instabot.pyV2#29]
Fill all requirements before sending issues. Or all issues will be closed without reading. 

- [ x] Did read [FAQ](https://github.com/ugurozturk/instabot.pyV2/wiki/FAQ)
- [ x] Searched and checked other issues about my problem
- [x] If have errors.log file and i did upload it

Check One :
- [ ] Bug
- [x] Feature
- [ ] Need Help

## What OS?
MAC / Windows / Linux(detail) / Android
## Python Version ?
2x / 3x
## Is it VPS ?
Yes / No
### VPS: Is the vps far far away ? Another Country ?
Yes / No
- [ ] VPS: Tried the same code via my local pc

## What is your python knowledge vote 0/10
--
The bot is indifferent to media types and comments **This is a nice photo** on videos.

**Comment by ugurozturk** [comment]
@sparshy if community thinks that is useful then i can make it. For now it doesn't sound good to me. Because you can make the comments to like "This is nice post" so it would work for photos and videos.

**Comment by sparshy** [comment]
Yeah man no worries, you already are doing much work. Ill try to implement it after my exam i.e. Feb 10. I am thinking on improving on the entire comment module as someone already mentioned these comments look very automatonlike.
